ci/test_python.sh

```diff
@@ -149,6 +149,13 @@ python -m nx_cugraph.scripts.print_tree --dispatch-name --plc --incomplete --dif
 python -m nx_cugraph.scripts.print_table
 popd
 
+rapids-logger "ensure nx-cugraph autogenerated files are up to date"
+pushd python/nx-cugraph
+make || true
+git diff --exit-code .
+git checkout .
+popd
+
```

Comment:
I like this check but we should do this as part of pre-commit like we're doing for the `rapids-dependency-file-generator` code gen. This will not only ensure the README is up-to-date on commit, but the same check you're doing here will be done as part of the [CI style checks](https://github.com/rapidsai/cugraph/blob/branch-24.04/ci/check_style.sh) automatically.  The CI style check is preferred because it fails much faster and has less log output to scroll through.

Reply:
No strong feelings from me, but adding this as a pre-commit slows down pre-commit for everybody. Could this be switched to a pre-commit hook in a different PR?

Reply:
Yeah that's also an option that seems better than performing the check as part of a test step.  I can do this in an upcoming PR I still need to do to fix the supported python version and CUDA version.